openeo/internal/process_graph_visitor.py

```diff
@@ -15,8 +15,9 @@ class ProcessGraphVisitor(ABC):
     Hierarchical Visitor for (nested) process graphs structures.
     """
 
-    def __init__(self):
+    def __init__(self, parameters = None):
         self.process_stack = []
+        self.parameters = parameters or {}
```

Comment:
@soxofaan this is self.parameters right?

Reply:
ah ok, I guess I was just looking at the last commit of the PR, instead of the whole diff (happens when you click through from a github notification email)